support/power-platform/power-automate/flow-approval-cds-provisioning-errors.md

```diff
@@ -17,67 +17,67 @@ _Original KB number:_ &nbsp; 4513672
 
 > Failed to create the Dataverse database in this environment with status code 'AADApplicationDisabled'.
 
 > Resource '`https://publishers.crm.dynamics.com`' has been disabled by your tenant administrator. Contact your tenant administrator and request that they enable '`https://publishers.crm.dynamics.com`' in the Azure Portal.'.
 
-These errors occur if the Dynamics CRM Online / Dataverse applications are disabled either in the tenant, or through Conditional Access for specific users. The exact error message may vary depending on the exact state of the Dataverse instance corresponding to the Power Apps/Flow environment - unprovisioned, provisioned but no Approvals installed, or Approvals already installed.
+These errors occur if the Dynamics CRM Online or Dataverse applications are disabled either in the tenant or through Conditional Access for specific users. The exact error message may vary depending on the exact state of the Dataverse instance corresponding to the Power Apps or Power Automate environment - unprovisioned, provisioned but no approvals installed, or approvals already installed.
 
-To resolve, tenant administrators will need to go to the **Enterprise Applications** tab under **Azure Active Directory** on [Microsoft Azure](https://ms.portal.azure.com) to ensure application 00000007-0000-0000-c000-000000000000 (Dataverse / Dynamics CRM Online) is enabled for users to sign in, and any relevant Conditional Access policies grant the necessary access to users expecting to use Flow Approvals.
+To resolve this issue, tenant administrators need to go to the **Enterprise Applications** tab under **Azure Active Directory** in [Microsoft Azure](https://ms.portal.azure.com) to ensure application 00000007-0000-0000-c000-000000000000 (Dataverse or Dynamics CRM Online) is enabled for users to sign in, and any relevant Conditional Access policies grant the necessary access to users expecting to use Power Automate Approvals.
 
-:::image type="content" source="media/flow-approval-cds-provisioning-errors/properties-settings.png" alt-text="Screenshot to set the Enabled for users to sign-in option to Yes.":::
+:::image type="content" source="media/flow-approval-cds-provisioning-errors/properties-settings.png" alt-text="Screenshot that shows how to set the Enabled for users to sign-in option to Yes.":::
 
 ## Error "Database is disabled"
 
 > The Dataverse database for this environment is disabled
 
-The Dataverse instance has been disabled in this environment. It isn't expected, and is related to the expiration of all Power Automate & Dataverse plans within your AAD tenancy. To ensure the database can be enabled, make sure at least one user has active plans.
+The Dataverse instance has been disabled in this environment. It isn't expected and is related to the expiration of all Power Automate & Dataverse plans within your Azure Active Directory (AD) tenancy. To ensure the database can be enabled, make sure at least one user has active plans.
```

Comment:
Minor nit. Should this be AAD not AD?

Reply:
Hi @hamenon-ms, I updated "Azure Active Directory (AD)" as "Azure Active Directory (Azure AD)". Sorry for the confusion.